server/services/utils/formValidation.js

```diff
@@ -11,6 +11,8 @@ const {
     lastItem
 } = require('../../utils/functionalHelpers');
 
+const proposedAddressConfig = require('../../routes/config/proposedAddress');
```

Comment:
Not really happy with accessing config directly, but otherwise the routes/review would need every config file just to pass it through